### PR TITLE
Add CardBet

### PR DESCRIPTION
**Pull request by Diogomartf**
- Add loading with LoadingCardBet
- data is just for mocking

image

### CODE REVIEW

**Review by berteotti — COMMENTED** [review]
Looking good, only missing a good mobile view

image

**Comment by berteotti** [comment]
Also, should we have gradients already at this stage?

**Comment by Diogomartf** [comment]
- Will improve mobile.
- I can add the gradients as well.

File: app/components/CardBet.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <Card className="w-full">
+      <Link key={market.id} href={`markets/${market.id}`} className="block">
+        <section className="p-4 h-[144px] flex flex-col justify-between space-y-4">
+          <div className="flex justify-between items-center">
+            <div className="flex space-x-2">
+              <Tag colorScheme="quaternary" size="sm">
+                Election
+              </Tag>
+              <Tag colorScheme="success" size="sm">
+                You choose Yes
```
Comment (berteotti) [inline app/components/CardBet.tsx:26]: Just to have a good example, not necessary at this stage but I prefer to have it right from the start.
We should use the past tense here.

```suggestion
                You chose Yes
```

File: app/my-bets/page.tsx [hunk 2]
```diff
@@ -1,17 +1,75 @@
...
+              <TabStyled>
+                All bets{" "}
+                <div className="bg-surface-surface-0 text-2xs border border-outline-low-em rounded-6 p-1 px-1.5 ml-2">
+                  09
+                </div>
+              </TabStyled>
+              <TabStyled>Active</TabStyled>
+              <TabStyled>Unredeemed</TabStyled>
+              <TabStyled>Complete</TabStyled>
+            </TabHeader>
```
Comment (berteotti) [inline app/my-bets/page.tsx:44]: This should be scrollable on mobile. Otherwise it will break the window size.

**Review by berteotti — APPROVED** [review]

**Pull request merged**